extensions/adobe/experience/adcloud/search/account-all.schema.json

```diff
@@ -0,0 +1,65 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/search/account-all",
+  "title": "Adobe AdCloud Search Account Entity Full Extension",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/profile-adcloud"
+  ],
+  "description": "Adobe AdCloud Search Account Entity Full Extension. Contains all Solution added fields.",
+  "definitions": {
+    "account-all": {
+      "properties": {
+        "account": {
+          "title": "Adobe Ad Cloud Search Account Top Level Fields",
+          "$ref": "https://ns.adobe.com/experience/adcloud/search/advertisingattributes",
+          "description": "Adobe Ad Cloud Top Level Entity Fields."
+        },
+        "currency": {
+          "title": "Currency the account is operated in",
+          "type": "string",
+          "description": "Currency associated with the account"
+        },
+        "managed": {
+          "title": "Is account managed or self serve",
```

Comment:
Why is this a string and not a boolean?

Reply:
This can be a boolean flag as there are only 2 possible values - managed and not managed. However many internal systems filter data based on these values rather than boolean flag. We will make this as an enum